pkg/sfu/forwarder.go

```diff
@@ -1609,7 +1610,8 @@ func (f *Forwarder) getTranslationParamsVideo(extPkt *buffer.ExtPacket, layer in
 		}
 	}
 
-	if f.currentLayers.Spatial != layer {
+	// if we have layer selector, let it decide whether to drop or not
+	if f.ddLayerSelector == nil && f.currentLayers.Spatial != layer {
```

Comment:
This is the fix for the forwarding which I broke with opportunistic forwarding changes.